content/assignments/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Assignments"
+date: 2023-01-03T10:42:56-06:00
+draft: false
+weight: 100
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## What is a Graded Assignment?
+
+Graded Assignments will count towards your final grade in the course. While Assignment 0 is more of a tutorial to get you used to autograding in C#, the four TechJobs assignments are required for you to move forward in Liftoff. Each assignment builds on concepts introduced over multiple 
+
```

Comment:
Paragraph is incomplete.  Would clarify that one must pass each graded assignment (Pass / Fail, not an actual grade) to graduate from the course continue into liftoff.

Reply:
Fixed!